cypress/integration/login/login.spec.js

```diff
@@ -1,7 +1,34 @@
 describe("Login", () => {
-  const username = Cypress.env("username");
-  const password = Cypress.env("password");
-  it("opens up the login page", () => {
-    cy.Login({ username, password })
+  it("logs in the default user", () => {
+    const user = Cypress.env("users").default;
+    cy.login(user);
+    cy.contains('Hello, there!');
+    cy.contains('Dashboard')
+    cy.get('ul.navbar-nav').should('not.contain', 'CRM'); // default user should not CRM
+    cy.get('ul.navbar-nav').should('not.contain', 'KnowledgeCafe'); // default user should not CRM
   });
+
+  it("logs in the employee user", () => {
+    const user = Cypress.env("users").employee;
+    cy.login(user);
+    cy.get("#wishlist").contains('Your wishlist');
+    cy.get('ul.navbar-nav').contains('CRM'); // employee should see CRM
+    cy.get('ul.navbar-nav').contains('KnowledgeCafe'); // employee should see KnowledgeCafe
+  });
+
+
+it("logs in the admin user", () => {
+  const user = Cypress.env("users").admin;
+  cy.login(user);
+  cy.get('ul.navbar-nav').contains('CRM'); // employee should see CRM
```

Comment:
the comment requires an update here. please update at other places also
```suggestion
  cy.get('ul.navbar-nav').contains('CRM'); // admin should see CRM
```

Reply:
done